export_assets_to_android_ios.jsx

```diff
@@ -1,10 +1,28 @@
-/**
+/*
 * Author: austynmahoney (https://github.com/austynmahoney)
 */
-var selectedExportOptions = {};
+
+Array.prototype.indexOf = Array.prototype.indexOf || function(value, start) {  
+  for (var i = 0, length = this.length; i < length; i++) {  
+    if (this[i] == value) {  
+      return i;  
+    }  
+  }  
+  return -1;  
+}
+
+var selectedExportOptions = {'ldpi': true, 'mdpi': true, 'hdpi': true, 'xhdpi': true, 'xxhdpi': true,'xxxhdpi': true};
+var options = {source: 100, size: 96};
+var selectedSizes = [12,16,24,32,36,48,60,72,94,96,106];
+var availableSizes = [12,16,24,32,36,48,60,72,94,96,106];
 
 var androidExportOptions = [
     {
+        name: "ldpi",
```

Comment:
I've kept `ldpi` out on purpose, the amount of phones using that image size are almost non-existent now.